static/app/components/performanceOnboarding/sidebar.tsx

```diff
@@ -4,8 +4,8 @@ import styled from '@emotion/styled';
 import HighlightTopRightPattern from 'sentry-images/pattern/highlight-top-right.svg';
 
 import Button from 'sentry/components/button';
-import DropdownMenuControlV2 from 'sentry/components/dropdownMenuControlV2';
-import {MenuItemProps} from 'sentry/components/dropdownMenuItemV2';
+import DropdownMenuControlV2 from 'sentry/components/dropdownMenuControl';
```

Comment:
should we update the name here too?
```suggestion
import DropdownMenuControl from 'sentry/components/dropdownMenuControl';
```